packages/server-wallet/src/models/finalization-status.ts

```diff
@@ -0,0 +1,70 @@
+import Knex from 'knex';
+import {Model} from 'objection';
+
+import {Bytes32, Uint48} from '../type-aliases';
+
+export type FinalizationStatusResult =
+  | {status: 'Finalized'; finalizedAt: number; finalizedBlockNumber: number}
+  | {status: 'Not Finalized'};
+
+interface RequiredColumns {
+  readonly channelId: Bytes32;
+  readonly finalizesAt: Uint48;
+  readonly blockNumber: Uint48;
+}
+export class FinalizationStatus extends Model implements RequiredColumns {
+  readonly channelId!: Bytes32;
+  readonly finalizesAt!: Uint48;
+  readonly blockNumber!: Uint48;
+  static tableName = 'finalization_status';
+
+  static async getFinalizationStatus(
+    knex: Knex,
+    channelId: Bytes32
+  ): Promise<FinalizationStatusResult> {
+    const result = await FinalizationStatus.query(knex)
+      .where({channelId})
+      .first();
+
+    return FinalizationStatus.convertResult(result);
+  }
+
+  static async updateFinalizationStatus(
+    knex: Knex,
+    channelId: string,
+    finalizesAt = 0,
```

Comment:
Should these have defaults?

I feel like setting `finalizesAt` to `0` should be a deliberate action, not the default action.